src/Runner.Listener/MessageListener.cs

```diff
@@ -58,8 +73,8 @@ public async Task<Boolean> CreateSessionAsync(CancellationToken token)
 
             // Create connection.
             Trace.Info("Loading Credentials");
-            var credMgr = HostContext.GetService<ICredentialManager>();
-            VssCredentials creds = credMgr.LoadCredentials();
+            _useMigratedCredentials = !StringUtil.ConvertToBoolean(Environment.GetEnvironmentVariable("GITHUB_ACTIONS_RUNNER_SPSAUTHURL"));
```

Comment:
What sets this env variable?

Reply:
End user.

Reply:
Is this something we will tell users to set to fix issues they are encountering then?